site/pages/docs/ref/texthighlight/texthighlight-en.hbs

```diff
@@ -26,6 +26,17 @@
 	</ul>
 </section>
 
+{{!-- Section for non-text contrast test.
+The ratio should be greater than 3:1.
+The background color used for highlight is #ffff00 and the font color is #000000, resulting in ratio of 19.55:1.
+--}}
+<section>
+	<h2>Non-text contrast</h2>
+	<p>According to <a href="https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html">Understanding Success Criterion 1.4.11: Non-text Contrast</a>, the contrast ratio is required to be at least 3:1.</p>
+	<p>The background color used for highlight is #ffff00 and the font color is #000000, resulting in ratio of 19.55:1</p>
+	<img src="../../img/contrast_ratio.png" alt="contrast_ratio">
```

Comment:
You do not need to put an image here. You can get rid of the image file.  Worst case you could put a link to your contrast checker if you really want to. Also, just as an FYI, `alt` attributes in `img` tags should be written in human readable language.

Apart from that, good to go.